files/containerd/config.toml

```diff
@@ -0,0 +1,253 @@
+### Managed by Puppet
+### generated with "containerd config default" on ubuntu 22.04
```

Comment:
it is also true for other os. just wanted to say where i got it from. when running `containerd config default` on centos stream 8 i get the same results. Thats the containerd standard config, except i changed `SystemdCgroup = false` to `true`.

when we don't set it, it will use the same config, because this are also the internal defaults.

Reply:
In that case, you should definitely cut the configuration down to only setting `SystemdCgroup`, hardcoding values to whatever defaults they were at that time is just guaranteed to lead to issues in the future, especially if you want to support multiple versions of containerd - which you're basically guaranteed to require as the module supports multiple distros.

Reply:
okay, can do.